### PR TITLE
Update the release automation for 1.8

### PR DESCRIPTION
**Pull request by kimwnasptd**
This PR will make the following changes:
1. Remove the logic of running the bundle tests before publishing a bundle
2. Update the scripts under `scripts/` to handle the 1.8 release
3. Copy the `tests-bundle/1.7` folder to `tests-bundle/1.8` folder

The reason, to our current understanding, that the tests are broken is because of https://github.com/canonical/oidc-gatekeeper-operator/issues/112. Once we fix that we should enable back the tests, that at least will verify that all components in the bundle become active

### CODE REVIEW

**Comment by kimwnasptd** [comment]
Note, my editor auto-updated the lint on the `scripts/get_releases_affected.py` file. That's why it shows it has many changes

File: tests-bundle/1.8/test_release_1-8.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        raise_on_error=False,
+        timeout=from_minutes(minutes=100),
+    )
+
+    print("dispatch istio config changed hook")
+    rc, stdout, stderr = await ops_test.run(
+        *shlex.split(
+            'juju run --unit istio-pilot/0 -- "export JUJU_DISPATCH_PATH=hooks/config-changed; ./dispatch"'
+        )
+    )
```
[the remark this answers is not in the document]
Comment (kimwnasptd) [reply tests-bundle/1.8/test_release_1-8.py:93]: No idea, I just copy pasted. But can completely remove it

File: scripts/get_releases_affected.py [hunk 2]
```diff
@@ -1,33 +1,36 @@
...
+    for file_path in sys.argv:
+        # check if string starts with the "releases"
+        file_path_starts_with_releases = re.search("^releases", file_path)
+
+        if file_path_starts_with_releases:
+            directories = file_path.split('/')
+            track = directories[1]
+            risk = directories[2]
+            accepted_tracks = ["1.4", "1.6", "1.7", "1.8", "latest"]
+            accepted_risks = ["beta", "edge", "stable"]
```
[the remark this answers is not in the document]
Comment (kimwnasptd) [reply scripts/get_releases_affected.py:20]: On it

File: scripts/get_releases_affected.py [hunk 3]
```diff
@@ -1,33 +1,36 @@
...
+
+    for file_path in sys.argv:
+        # check if string starts with the "releases"
+        file_path_starts_with_releases = re.search("^releases", file_path)
+
+        if file_path_starts_with_releases:
+            directories = file_path.split('/')
+            track = directories[1]
+            risk = directories[2]
+            accepted_tracks = ["1.4", "1.6", "1.7", "1.8", "latest"]
```
[the remark this answers is not in the document]
Comment (kimwnasptd) [reply scripts/get_releases_affected.py:19]: On it

[the remark this answers is not in the document]
Comment (kimwnasptd) [reply scripts/get_releases_affected.py]: Pushed a commit for this that:
1. Creates a global var at the top
2. Keeps only `1.7`, `1.8` and `latest` versions

File: scripts/get_releases_affected.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (kimwnasptd) [reply scripts/get_releases_affected.py]: Pushed a commit

File: tests-bundle/1.8/test_release_1-8.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (kimwnasptd) [reply tests-bundle/1.8/test_release_1-8.py]: Pushed a commit to remove the logic of triggering the `config-changed` event

**Comment by kimwnasptd** [comment]
@DnPlas thanks for the review! I addressed your comments

**Review by DnPlas — APPROVED** [review]
Thanks @kimwnasptd

**Pull request merged**